interfaces/udev/backend.go

```diff
@@ -59,7 +60,7 @@ func (b *Backend) Setup(snapInfo *snap.Info, devMode bool, repo *interfaces.Repo
 	if err != nil {
 		return fmt.Errorf("cannot obtain expected udev rules for snap %q: %s", snapName, err)
 	}
-	glob := fmt.Sprintf("70-%s.rules", interfaces.SecurityTagGlob(snapName))
+	glob := fmt.Sprintf("70-%s.rules", snap.SecurityTag(snapName))
```

Comment:
Lets do that if nothing else speaks against this. Wanted to have some initial comments on this if I am right or not.